docs/tasks/debug-application-cluster/audit.md

```diff
@@ -91,14 +109,62 @@ audit backend using the following kube-apiserver flags:
 
 - `--audit-webhook-config-file` specifies the path to a file with a webhook
   configuration. Webhook configuration is effectively a [kubeconfig][kubeconfig].
-- `--audit-webhook-mode` define the buffering strategy, one of the following:
-  - `batch` - buffer events and asynchronously send the set of events to the external service
-    This is the default
-  - `blocking` - block API server responses on sending each event to the external service
+- `--audit-webhook-initial-backoff` specifies the amount of time to wait after the first failed
+  request before retrying. From the second request and so on, exponential backoff is used.
```

Comment:
edit last sentence here to:
"Subsequent requests are retried with exponential backoff."
("and so on" indicates other types of events, not additional requests)

Reply:
Done, thanks